### ISSUE TITLE
Request: Ability to import JSON files for Settings configuration

### ISSUE THREAD

**Issue by 23Aaron** [Balackburn/Apollo#20]
We can export these settings but can't import them, so hopping between different patched IPAs can be a pain

**Comment by vbmark** [comment]
Yes, this is needed for people like me who have many accounts for different subjects.
